starlite_jwt/middleware.py

```diff
@@ -73,3 +79,78 @@ async def authenticate_request(self, connection: "HTTPConnection") -> Authentica
             raise NotAuthorizedException()
 
         return AuthenticationResult(user=user, auth=token)
+
+
+class CookieOptions(BaseModel):
+    path: str = "/"
+    domain: Optional[str] = None
+    secure: Optional[bool] = None
+    samesite: Literal["lax", "strict", "none"] = "lax"
+    description: Optional[str] = None
+
+
+class JWTCookieAuthenticationMiddleware(JWTAuthenticationMiddleware):
+    """JWT Cookie Authentication Middleware.
+
+    Args:
+        JWTCookieAuthenticationMiddleware (_type_): _description_
```

Comment:
I think this is. a mistake. 

Also - according to the style guide, class comments should be added either inside the `__init__` method OR top level, not both. I prefer using `__init__` for  consistency.

Reply:
Removed the docstrings.  Re: the mistake, was that in reference to the docstring or the something else?